tests/testthat/test-expect_lint.R

```diff
@@ -59,7 +59,23 @@ test_that("expect_lint_free works", {
   )
   withr::local_envvar(c(NOT_CRAN = "true", R_COVR = "false"))
 
-  expect_lint_free(test_path("dummy_packages", "clean"))
-  expect_lint_free(test_path("dummy_packages", "clean_subdir", "r"))
+  withr::with_dir(
+    test_path("dummy_packages", "clean"),
+    {
+      callr::rcmd("INSTALL", ".", echo = FALSE, show = FALSE)
```

Comment:
No need to skip if not installed because we already rely on `{callr}`.

```r
> pak::pkg_deps_explain("lintr", "callr")
✔ Updated metadata database: 4.80 MB in 10 files.                         
ℹ R 4.2 aarch64-apple-darwin20 packages are missing from Bioconductor      
✔ Updating metadata database ... done                                     
lintr -> cyclocomp -> callr 
```

Reply:
that may be true now but not guaranteed to hold forever. safer to keep it conditional as long as it's in our own Suggests

Reply:
Okay, running the tests conditionally now.